inc/auth.class.php

```diff
@@ -261,7 +261,7 @@ public static function set_user_token($user_login, $current_user)
         /* Send token */
         wp_mail(
             $user_email,
-            __('Dein Sicherheitscode'),
+            __('Your security code', '2-Step-Verification'),
```

Comment:
@00travelgirl00 You should use `esc_html__()` as function to translate a string. It is more secure if we haven't the control about the content inside the translation files. helpful: https://gist.github.com/bueltge/49c652187ef10a448bfb
Please can you change this inside the pull request, after that I will merge it. Thanks.